doc/docs/master/reference/scaling_limits.md

```diff
@@ -0,0 +1,58 @@
+
+Are you hitting those limits? You are likely to be needing our assistance in optimizing your DAGs. We are here to help! See [Lift the limitation](#lift-the-limitation) and request a free trial of our Enterprise Edition.
+
+## Lift the limitations
+
+To experiment with Pachyderm without limit, we provide an easy way to request an [**Enterprise Edition trial token**](https://www.pachyderm.com/trial). Fill in the form (link provided) and receive an enterprise license token.
+
+!!! Info "You might qualify for a free Enterprise license." 
+    Pachyderm offers activation keys for proofs-of-concept, startups, academic, nonprofit, or open-source projects. [Tell us about your project!](https://www.pachyderm.com/trial). 
+
+## What happens when you exceed those limits?
+
+As a general rule, Pachyderm provides a message in the STDERR whenever a limit is encountered.
+
+For example, each `pachctl` command invoking an enterprise feature (such as `pachctl auth,` `pachctl deploy ide`...) logs an alert message in your STDERR with a link to the [Enterprise Edition documentation](https://docs.pachyderm.com/latest/enterprise/) for more information. 
+
+### Limit on the number of pipelines
+When exceeding the number of pipelines:
+
+- `pachctl create pipeline` fails once the maximum number of pipelines is reached.
+
+- `pachctl update pipeline`  and `pachctl edit pipeline` succeed on existing pipelines, fail when attempting to create pipelines beyond the limit. 
+!!! Note
+    If `update pipeline` fails for any other reason, it does not log any message related to pipeline limits.
+
+- `pachctl restore` fails when trying to restore with too many pipelines. 
+
+All of the commands listed above create a distinct message to STDERR and to the pachd logs. This message includes information such as the limit on the number of pipelines in the Community Edition, the total number of pipelines deployed, and provides a link to request an Enterprise key to lift those limitations.
+
+- all other `extract`, `list`, `run`, `start`, `stop pipeline` commands' behavior remains unchanged.
+
+### Limit on the number of workers per pipeline
+Pachyderm has [two parallelism](https://docs.pachyderm.com/latest/reference/pipeline_spec/#parallelism-spec-optional) strategies: constant and coefficient.
+
+When `constant` parallelism > 8: 
+
+- `pachctl create pipeline` and `pachctl update pipeline` fail. A message to STDERR and pachd logs is generated. You will need to update your pipeline specification file accordingly or activate an Enterprise license.
+
+The previous behavior differs for a `coefficient` parallelism:
+
+- `pachctl create, update, edit pipeline` will succeed. However, Pachyderm will scale the number of workers up to the Community Edition limit, and ignore the remaining workers if the value set on `coefficient` results in more than 8 workers.  A message to STDERR and pachd logs is generated.
```

Comment:
It's more accurate to just say a message is logged to the pachd logs.  The stderr logging is for when the user is running the command, where we need to give feedback on why a command failed.